UnityProject/Assets/Scripts/SpriteHandler/SpriteHandler.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Networking;
+
+public class SpriteHandler : NetworkBehaviour
+{
+	public SpriteRenderer spriteRenderer;
+
+	public List<SpriteList> spriteList = new List<SpriteList>();
```

Comment:
structure makes sense to me in editor, but might not to others. If you rename this to animationList I think it will help. You have a list of animations, and each animation has a list of sprites. So calling this a spriteList confuses between what an animation is and what a sprite is.